tools/cli/commands/phan.js

```diff
@@ -662,4 +667,16 @@ export async function handler( argv ) {
 		reportStream.end();
 		console.log( `Report written to ${ argv.reportFile }` );
 	}
+	if ( ( process.exitCode & 2 ) !== 0 ) {
```

Comment:
What exit codes are possible? This would account for 1 and 2, but if 4 (or 5, or 8...) are possible weird error codes, wouldn't we want to show the "try -v" message?

Reply:
See line 499 above. The possible codes are:

0: Phan ran successfully, no issues found.
1: Phan ran successfully, issues found.
2: Phan failed to run for at least one project. No issues found in any that did run.
3: Phan failed to run for at least one project, and issues were found in some that did run.

Reply:
I saw 1 and 2, but didn't see 0 (it was undefined when I was looking, which works okay) or 3.

Reply:
To get 3, you have to do both of these tests at the same time in different projects, and run phan on both of the projects at once:

* Add `echo FAIL; exit 1` to `projects/packages/status/.phan/pre-run`
* Introduce a Phan issue (e.g. delete a suppression from `.phan/baseline.php`)

Reply:
Ah, perfect, thanks!